### PR TITLE
ocamlPackages.ezxmlm: init at 1.0.2

### PR DESCRIPTION
**Pull request by carlosdagos**
###### Motivation for this change

Nice library to have :)

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by srhb** [comment]
@GrahamcOfBorg build ocamlPackages.ezxmlm

**Comment by srhb** [comment]
Are you sure this is the MIT license? It looks more like a (mangled...) version of the OpenBSD license to me.

**Comment by carlosdagos** [comment]
@srhb well spotted. I’m actually not sure 😅 I now remember I needed to look that up. The license file in the repo is a bit weird indeed. Do you reckon we can not include that metadata or is there some default we can use like “unknown”? Thanks for reviewing

**Comment by srhb** [comment]
I think it's only the formatting that's mangled, but perhaps you can confirm with a proofread: https://en.wikipedia.org/wiki/ISC_license#OpenBSD_license

If that is indeed the case, I think we should just add the openbsd license and refer to that.

**Comment by carlosdagos** [comment]
Updated in the latest commit :)

File: pkgs/development/ocaml-modules/ezxmlm/default.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+stdenv.mkDerivation rec {
+  version = "1.0.2";
+  name = "ocaml${ocaml.version}-ezxmlm-${version}";
+
+  src = fetchFromGitHub {
+    owner = "avsm";
+    repo = "ezxmlm";
+    rev = "v${version}";
+    sha256 = "1dgr61f0hymywikn67inq908x5adrzl3fjx3v14l9k46x7kkacl9";
+  };
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/ezxmlm/default.nix]: Maybe the release artifact (https://github.com/avsm/ezxmlm/releases/download/v1.0.2/ezxmlm-1.0.2.tbz) is a better source.

Comment (carlosdagos) [reply pkgs/development/ocaml-modules/ezxmlm/default.nix]: Tried using this and it [requires](https://github.com/avsm/ezxmlm/blob/master/pkg/pkg.ml#L3) [`topkg-jbuilder`](https://github.com/samoht/topkg-jbuilder), which I don't think is currently available in nixpkgs. Don't know if I'm missing something though :)

**Comment by carlosdagos** [comment]
Thanks for reviewing @vbgl. See latest commit with the changes you requested. Only thing what I couldn't get to work is using the release artifact, so still using `fetchFromGithub`.

**Pull request merged**